blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md

```diff
@@ -0,0 +1,129 @@
+| **Total**         | **$15.70**        | **100.0%**            | **$33.80**       | **100.0%**            | **16.6%**          |
+
+*Source: IDC Worldwide Open Compute Project Compute and Storage Infrastructure Market Forecast, May 2020.*
+
+The OCP Open System Firmware project is aiming to allow system firmware like
+[BIOS][3] and [BMC][4] to be modified and their source code shared with the
+owners of OCP-certified hardware. Since March 2021, supporting Open System
+Firmware is mandatory for servers with OCP badging. The ecosystem partners have
+decided to implement such Open Source Firmware (OSF) as coreboot for their Open
+System Firmware architecture, since having the source code helps to adapt the
+firmware to the needs of even the most demanding clients by fixing the firmware
+bugs and expanding the device's feature set - at a reduced cost - which allows
+making the most out of both your existing and future products by extending their
+lifecycle and bringing them to the new previously unthought-of markets.
+
+![Embedded software market](/img/Embedded_Software_Market.png)
+
+Just being open-source - by itself is not enough for the firmware to meet the
+OCP certification: the extra requirements are mentioned in a [OSF Checklist][5].
+The source code should be easily buildable by a native (ideally the open-source)
+toolchain and the flashing utility should be either open-source like [flashrom][6]
+or at least documented well enough for an open-source implementation to be
+written. The key Open System Firmware components have to be open-source, and
+although it is allowed to include some closed-source ones *(aka binary blobs)* to
+Open System Firmware - there should be a valid justification for them to be
+approved, such as "containing silicon IP": i.e. a blob has been provided by your
+SV (Silicon Vendor) like Intel to set up their chip during boot, which is needed
+for your device to run. So the Open System Firmware has to be as close to being
+the Open Source Firmware as it is possible. And all the documentation should be
+good enough and distributable without NDAs.
+
+![OSF features](/img/OSF_features.png)
+
+The diagram above shows 6 types of Open Source Firmware features: the
+fundamental features to the more advanced ones are depicted from left to right
+and 1st to 3rd are essential.
+
+1. Chip Initialization and Integration: the firmware components for the initial
+setup of the motherboard's key chips - like the chips themselves - are usually
+provided by the Silicon Vendors, sometimes in a binary-only form *(without the*
+*source code)* like Intel FSP and later AMD AGESA. We at [3mdeb][7] can audit the
+source code either available publicly or accessible to us thanks to our NDAs &
+close cooperative relationships with Silicon Vendors like Intel and AMD. When
+the sources aren't available, we can reverse-engineer a binary-only component to
```

Comment:
Should we announce reverse-engineering of the blobs which license is against reverse engineering?

Reply:
@miczyg1 We don't announce there which blobs in particular we can reverse-engineer, so I think this is fine.

Reply:
@miczyg1 @mikebdp2 this is fine.